client/src/components/BaseComponents/GModal.vue

```diff
@@ -33,6 +33,8 @@ const props = defineProps<{
     title?: string;
     /** Fixes the height of the modal to a pre-set height based on `size` */
     fixedHeight?: boolean;
+    /** When false, keeps the modal open on "ok" */
+    closeOnOk?: boolean;
```

Comment:
I think this should be `true` by default to avoid breaking existing usages.

Reply:
it is. I can make that more explicit in code

Reply:
https://github.com/galaxyproject/galaxy/pull/20418/commits/daec7e1e5ee29a5ab206b7563cf44fd24d83d8e6 fixed it, thank you!